packages/xo-web/src/common/xo/index.js

```diff
@@ -495,12 +495,11 @@ subscribeCheckSrCurrentState.forceRefresh = pool => {
 const missingPatchesByHost = {}
 export const subscribeHostMissingPatches = (host, cb) => {
   const hostId = resolveId(host)
-
-  if (missingPatchesByHost[hostId] == null) {
-    missingPatchesByHost[hostId] = createSubscription(() =>
-      getHostMissingPatches(host)
-    )
-  }
+  host.power_state !== 'Running'
+    ? (missingPatchesByHost[hostId] = () => [])
```

Comment:
You can return a `noop` unsubscribe function because this solution will break this [code](https://github.com/vatesfr/xen-orchestra/blob/1af751c7c29deece7f9d984c853945729d1e7aa1/packages/xo-web/src/common/xo/index.js#L514).